lib/optimizely_server_side/optimizely_sdk.rb

```diff
@@ -11,7 +11,11 @@ class << self
       # Datafile
       def project_instance(options = {})
         Optimizely::Project.new(cached_datafile,
-                                options[:event_dispatcher])
+                                options[:event_dispatcher],
+                                nil,
+                                nil,
+                                skip_json_validation = true
```

Comment:
yeah, let's do that. I somehow wanted to knw in future what this true means . Will added a comment 👍